### PR TITLE
Feature/hds timeinput

### PR DESCRIPTION
**Pull request by minevala**
- Use HDS TimeInput component in opening hours inputs to improve usability
Näyttökuva 2021-5-28 kello 16 52 28

### CODE REVIEW

File: src/opening-period/time-span/TimeSpan.tsx [hunk 1]
```diff
@@ -32,6 +31,26 @@ const descriptionPlaceholderTexts: LanguageStrings = {
   en: 'Esim. for women only',
 };
 
+const validateTimeFormat = (timeString: string): boolean => {
+  const [hours, minutes] = timeString.split(':');
+  return (
+    (!hours && !minutes) ||
```
Comment (VilleMiekkoja) [inline src/opening-period/time-span/TimeSpan.tsx:37]: Does this mean that the time format is valid if there's no hours and no minutes?

**Comment by VilleMiekkoja** [comment]
Good work 👍

File: src/opening-period/time-span/TimeSpan.tsx (hunk 1 above)
Comment (minevala) [reply src/opening-period/time-span/TimeSpan.tsx:37]: I could rename this or something. I agree, the name does not make sense 👋

**Review by VilleMiekkoja — APPROVED** [review]
Nice 👍

**Pull request merged**